src/Arriba/Arriba.Server/Startup.cs

```diff
@@ -63,7 +61,7 @@ public void Configure(IApplicationBuilder app, IHostEnvironment env)
             app.UseRouting();
             app.UseCors();
             app.UseArribaExceptionMiddleware();
-
+            
```

Comment:
Some /t and spaces where added, I do try to use <CTRL>+KD at VS to remove it for me.. It may going to impact the change above.

Reply:
Likely due to a rebase merge. I'll clean it up. We should add a build step to break on mixed mode files.